launch/tier4_localization_launch/launch/eagleye/fix2pose.xml

```diff
@@ -0,0 +1,36 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<launch>
+  <arg name="output_pose_with_cov_name" default="/localization/pose_estimator/pose_with_covariance"/>
+  <node pkg="eagleye_fix2pose" name="fix2pose_node" exec="fix2pose">
+    <!-- /eagleye/fix or  /eagleye/rtk_fix(rtk_dead_reckoning mode) or /GNSS/fix(GNSS pose only mode)-->
+    <remap from="eagleye/fix" to="eagleye/fix" />
+    <remap from="ekf_trigger_node" to="/localization/pose_twist_fusion_filter/trigger_node"/>
```

Comment:
You should change the name "ekf_trigger_node".

Reply:
Why is that? ...

Reply:
Sorry, I did not understand it well enough.
This part is fine as is.